libpod/networking_linux.go

```diff
@@ -62,7 +63,7 @@ const (
 // This is need because a HostIP of 127.0.0.1 would now allow the gvproxy forwarder to reach to open ports.
 // For machine the HostIP must only be used by gvproxy and never in the VM.
 func (c *Container) convertPortMappings() []types.PortMapping {
-	if !c.runtime.config.Engine.MachineEnabled || len(c.config.PortMappings) == 0 {
+	if !machine.IsPodmanMachine() || len(c.config.PortMappings) == 0 {
```

Comment:
must use IsGvProxyBased() see the code comment for the function, unless there is a reason why we should ignore the host ip on wsl as well?

Reply:
I added it this way since would be helpful to have something similar for WSL as well, since the user cam refer to interfaces that do not exist on the system (e.g. physical ethernet address on the windows host). Although it is less of a problem (since 127.0.0.1 works), and it does prevent some workarounds from being specific so on second thought I will set to qemu only for now and revisit later.